demo_app/build.gradle

```diff
@@ -27,5 +27,5 @@ android {
 
 dependencies {
     implementation project(':snackengage-playrate')
-    implementation 'com.android.support:appcompat-v7:28.0.0'
+    implementation 'androidx.appcompat:appcompat:1.0.0'
```

Comment:
why not directly appcompat 1.1.0?

Reply:
As I understand the `appcompat-v7:28.0.0` contains the same code as `appcompat:1.0.0` aside from the package names. I therefore would release this "carbon copy" as a new library version to make the migration for app project seemless. Afterwards, I would increase the versions of these dependencies and release another library version. App authors can then easily pick the library version which makes the transition for them most pleasant.

Reply:
Now I see - makes sense - good thinking!